brigade/templates/events.html

```diff
@@ -44,16 +44,18 @@ <h2 class="slab__title">Key Dates</h2>
         <li>
           <h3>
             <small>August 11, 2018</small> <br>
-            <a href="http://nationaldayofcivichacking.org" target="_blank"><strong>National Day of Civic Hacking</strong> <i class="fas fa-external-link-alt fa-xs"></i></a></a>
+            <a href="http://nationaldayofcivichacking.org" target="_blank"><strong>National Day of Civic Hacking</strong> <i class="fas fa-external-link-alt fa-xs"></i></a>
           </h3>
           <p>
             This annual event brings together public servants, people with technology skills, and community organizers, to show that government can work in the 21<sup>st</sup> century if we all build it together.
           </p>
         </li>
         <li>
-          <h3><small>October 19–21, 2018</small> <br><strong>Brigade Congress</strong></h3>
+          <h3><small>October 19–21, 2018</small> <br>
+            <a href="https://www.codeforamerica.org/brigade-congress" target="_blank"><strong>Brigade Congress</strong> <i class="fas fa-external-link-alt fa-xs"></i></a>
+          </h3>
           <p>
-            Brigade Congress is an annual unconference where brigade participants share stories on what has worked, what hasn't, and how we can apply those lessons to future wins. (More details coming soon!)
+            Brigade Congress is the primary in-person meeting centered on the Code for America Network–Brigade leaders, government and community partners, and civic tech professionals. Join us in Charlotte, North Carolina for this unique knowledge-sharing and training event.
```

Comment:
The first few times I read this line, I was reading the em-dash as a hyphen, as in 'Network-Brigade leaders'. Is there a way we can rewrite this sentence without the em-dash? Maybe something like:
> Brigade Congress is the primary in-person meeting centered on the Code for America Network, bringing together Brigade leaders, government and community partners, and civic tech professionals from across the country.

Reply:
Good call, I've updated it to your suggestion.